scripts/multisigTransactionProposals/README.md

```diff
@@ -0,0 +1,85 @@
+# Proposing Multisig Transactions via the Safe SDK
```

Comment:
Could we put `multisigTransactionProposals` into the contracts directory instead? We will likely at some point make the contracts directory its own repo.